pkg/grpc/client.go

```diff
@@ -85,30 +86,31 @@ func (c *Client) Request(serviceMethod string, message string, headers []string)
 	})
 
 	if connErr != nil {
-		log.Printf("grpc client connect: %v", connErr)
-		return response.Response{Duration: time.Duration(0), Err: connErr, RequestSent: false, Type: "grpc"}
```

Comment:
No longer capturing the number of requests sent/not sent since it's much more clear to look at logs and see what happened during the warmup.

![image](https://user-images.githubusercontent.com/1187103/82729865-0a2e1680-9cf3-11ea-98d5-894e612702ae.png)